apps/admin-server/src/pages/projects/[project]/widgets/rawresource/[id]/styling.tsx

```diff
@@ -0,0 +1,189 @@
+    { label: 'Margin top 0', value: 'mt-0' },
+    { label: 'Margin bottom 0', value: 'mb-0' },
+    { label: 'Margin top and bottom s', value: 'my-2' },
+    { label: 'Margin top and bottom m', value: 'my-4' },
+    { label: 'Margin top and bottom lg', value: 'my-6' },
+    { label: 'Padding full s', value: 'p-2' },
+    { label: 'Padding full m', value: 'p-4' },
+    { label: 'Padding full lg', value: 'p-6' },
+    { label: 'Padding top and bottom s', value: 'py-2' },
+    { label: 'Padding top and bottom m', value: 'py-4' },
+    { label: 'Padding top and bottom lg', value: 'py-6' },
+  ],
+  Layout: [
+    { label: 'Display block', value: 'block' },
+    { label: 'Display flex', value: 'flex' },
+    {
+      label: 'Center items (needs display flex & (max) width)',
+      value: 'justify-center items-center',
+    },
+    { label: 'Center container (needs (max) width)', value: 'mx-auto my-auto' },
+    { label: 'Text align left', value: 'text-left' },
+    { label: 'Text align center', value: 'text-center' },
+    { label: 'Text align right', value: 'text-right' },
+  ],
+
+  Box: [
+    { label: 'Rounded corners xs', value: 'rounded-xs' },
+    { label: 'Rounded corners sm', value: 'rounded-sm' },
+    { label: 'Rounded corners md', value: 'rounded-md' },
+    { label: 'Rounded corners lg', value: 'rounded-lg' },
+    { label: 'Rounded corners full', value: 'rounded-full' },
+    { label: 'Border thin grey', value: 'border-2 border-grey' },
+    { label: 'Border thin blue', value: 'border-2 border-blue' },
+    { label: 'Border thin black', value: 'border-2 border-black' },
+    { label: 'Box shadow s', value: 'shadow-sm' },
+    { label: 'Box shadow m', value: 'shadow-md' },
+  ],
+  Typografie: [
+    { label: 'Font italic', value: 'italic' },
+    { label: 'Font bold', value: 'font-bold' },
+    { label: 'Text underline', value: 'underline' },
+    { label: 'Font size xs', value: 'text-xs' },
+    { label: 'Font size s', value: 'text-sm' },
+    { label: 'Font size m', value: 'text-md' },
+    { label: 'Font size lg', value: 'text-lg' },
+    { label: 'Font size xl', value: 'text-xl' },
+    { label: 'Text color white', value: 'text-white' },
+    { label: 'Text color black', value: 'text-black' },
+  ],
+  Sizing: [
+    { label: 'Max width sm', value: 'max-w-sm' },
+    { label: 'Max width md', value: 'max-w-md' },
+    { label: 'Max width lg', value: 'max-w-lg' },
+    { label: 'Max width xl', value: 'max-w-xl' },
+    { label: 'Max width 2xl', value: 'max-w-2xl' },
+  ],
+  Overig: [{ label: 'Overflow hidden', value: 'overflow-hidden' }],
+};
+
+const formSchema = z.object({
+  stylingClasses: z.array(
+    z.object({
+      label: z.string(),
+      value: z.string(),
+    })
+  ),
+});
+
+export default function WidgetRawStylingClasses(
+  props: RawResourceWidgetProps & EditFieldProps<RawResourceWidgetProps>
+) {
+  type FormData = z.infer<typeof formSchema>;
+  const { onFieldChange } = useFieldDebounce(props.onFieldChanged);
```

Comment:
Voor selects en checkboxes gebruik geen useIeldDebounce hook omdat de actie zo kort is dat de debounce timer alleen maar voor latency zorgt